samples/crypto/ecdsa/README.rst

```diff
@@ -7,23 +7,27 @@ Crypto: ECDSA
    :local:
    :depth: 2
 
+The ECDSA sample shows how to sign and verify messages using SHA-256 as the hashing algorithm and the secp256r1 curve.
+
 Overview
 ********
 
-The ECDSA sample shows how to sign/verify messages using SHA256 as the hashing algorithm and the secp256r1 curve.
+The sample follows these steps:
+
+1. First, the sample performs initialization:
 
+   a. The Platform Security Architecture (PSA) API is initialized.
+   #. A random ECDSA key pair is generated and imported into the PSA crypto keystore.
```

Comment:
Rename it to ECC key pair (please write it all out to help with the "unknown" acronym)

@Vge0rge: Is this present in the code as well?

Reply:
Indeed it is used in the code as well. So my recommendation is the same as before.

Reply:
Changed to ECC